Assets/MixedRealityToolkit.Tests/TestUtilities.cs

```diff
@@ -113,6 +112,20 @@ public static void InitializePlayspace()
             });
         }
 
+        /// <summary>
+        /// Forces the playspace camera to face forward.
+        /// </summary>
+        public static void FaceCameraForward()
```

Comment:
Hm...this method does something really similar to the one above, but has a very different name. Perhaps this method should be called "PlayspaceToOriginLookingForward"  and the method above shoudl be called "PlayspaceTCornerLookingAtOrigin" or something like that. Trying to brainstorm more descriptive method names.